modules/@apostrophecms/area/ui/apos/components/AposBreadcrumbSwitch.vue

```diff
@@ -97,60 +137,76 @@ export default {
     box-shadow: 0 0 0 4px #b5c9fc;
   }
 
-  div {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: center;
+  &--disabled {
+    .apos-breadcrumb-switch__input:not(:checked) + .apos-breadcrumb-switch__input-text:hover {
+      background-color: inherit;
+    }
 
-    label:first-child span {
-      left: -1px;
+    .apos-breadcrumb-switch__input:not(:checked) + .apos-breadcrumb-switch__input-text {
+      background-color: var(--a-white);
+      border: none;
+      box-shadow: none;
     }
 
-    label:last-child span {
-      right: -1px;
+    .apos-breadcrumb-switch__input:checked + .apos-breadcrumb-switch__input-text {
+      box-shadow: 0 0 0 1px var(--a-base-5);
+      background-color: var(--a-base-5);
+      color: var(--a-text-primary);
     }
-  }
 
-  input[type="radio"] {
-    position: absolute;
-    overflow: hidden;
-    width: 1px;
-    height: 1px;
-    clip: rect(0 0 0 0);
-    clip-path: inset(100%);
-    white-space: nowrap;
-
-    &:checked + span {
-      box-shadow: 0 0 0 1px var(--a-primary-dark-15);
-      background-color: var(--a-primary-transparent-90);
-      z-index: $z-index-default;
-      color: var(--a-text-inverted);
+    .apos-breadcrumb-switch__label {
+      opacity: 0.5
     }
   }
+}
 
-  label {
-    display: flex;
-    align-items: center;
+.apos-breadcrumb-switch__label {
+  display: flex;
+  align-items: center;
 
-    &:hover input:not(:checked) + span {
-      background-color: var(--a-base-9);
-    }
+  &:first-child .apos-breadcrumb-switch__input-text {
+    left: -1px;
+  }
 
-    span {
-      position: relative;
-      display: inline-flex;
-      box-sizing: border-box;
-      align-items: center;
-      height: 100%;
-      margin-left: .063em;
-      padding: 0 8px;
-      color: var(--a-base-1);
-      text-align: center;
-      transition: background-color 500ms ease;
-      cursor: pointer;
-      letter-spacing: 0.5px;
-      border-radius: 4px;
-    }
+  &:last-child .apos-breadcrumb-switch__input-text {
+    right: -1px;
+  }
+}
+
+  .apos-breadcrumb-switch__input:not(:checked) + .apos-breadcrumb-switch__input-text:hover {
+    background-color: var(--a-base-9);
   }
+
+.apos-breadcrumb-switch__input {
+  position: absolute;
+  overflow: hidden;
+  width: 1px;
+  height: 1px;
+  clip: rect(0 0 0 0);
+  clip-path: inset(100%);
+  white-space: nowrap;
+
+  &:checked + .apos-breadcrumb-switch__input-text {
+    box-shadow: 0 0 0 1px var(--a-primary-dark-15);
+    background-color: var(--a-primary-transparent-90);
+    z-index: $z-index-default;
+    color: var(--a-text-inverted);
+  }
+}
+
+.apos-breadcrumb-switch__input-text {
+  position: relative;
+  display: inline-flex;
+  box-sizing: border-box;
+  align-items: center;
+  height: 100%;
+  margin-left: .063em;
+  padding: 0 8px;
+  color: var(--a-base-1);
+  text-align: center;
+  transition: background-color 500ms ease;
+  cursor: pointer;
+  letter-spacing: 0.5px;
+  border-radius: 4px;
 }
```

Comment:
This css is jsut about using classes and adding @stuartromanek style when disabled.